CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added configurable logging macros to stm32 platform
 - Added support for ULP wakeup on ESP32
 - Added heap growth strategies as a fine-tuning option to `spawn_opt/2,4`
+- Added `crypto:crypto_one_time/4,5` on ESP32
```

Comment:
This needs a few other things:

1. A function definition in `crypto.erl` with a type spec that passes dialyzer.  Doing this will also result in the function being added to the on-line documentation.
2. An entry in the programmer’s guide explaining what the function does.

Also, would you consider an implementation for generic_unix that uses OpenSSL APIs?  This would give you a good opportunity to test against API compatibility with the OTP implementation.

Reply:
1. Sounds like a good idea.
2. Explaining crypto stuff is hard and risky, I would reference Erlang documentation and add maybe just a note about what is supported / not supported.

An OpenSSL based implementation sounds like a good idea, but quite a big effort that I would postpone with another PR.

Reply:
Regarding (3), a generic unix implementation could be tested against the OTP implementation but unfortunately we don't run these tests with esp32 port (or any other port for that matter). We need to fix that. Meanwhile, I did run the test manually against OTP26 (and obviously it passed).

Reply:
Fixed 1. and 2.